C/CMakeWrapper/Compat.toml

```diff
@@ -12,5 +12,5 @@ julia = "0.6-0"
 ["0.2-0"]
 julia = "0.7-1"
 
-["0.2.2-0"]
+["0.2.2"]
 CMake = "1"
```

Comment:
Keep this requirement?

Reply:
bump @rdeits

Reply:
Done (I think--this whole format is still a bit new to me)